src/mono/netcore/System.Private.CoreLib/src/Mono/Console.Mono.cs

```diff
@@ -10,10 +10,10 @@ namespace Internal
     // Some CoreCLR tests use it for internal printf-style debugging in System.Private.CoreLib
     public static class Console
     {
-        public static void Write(string? s) => DebugProvider.WriteCore(s);
+        public static void Write(string? s) => DebugProvider.WriteCore(s ?? string.Empty);
```

Comment:
FYI, this is a functional change to match coreclr's behavior.  This is also just internal debug code, though, anyway.